src/components/CustomTextArea/CustomTextArea.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { TextArea } from 'grommet';
+
+const CustomTextArea = ({ ...rest }) => <TextArea {...rest} />;
```

Comment:
Please use Context API. Don't spread properties through the whole tree. It is deprecated.

Reply:
Why Context API? I want to extend grommet component so I need other props